pkg/controller/certificates/issuing/issuing_controller.go

```diff
@@ -157,8 +158,7 @@ func NewController(
 }
 
 func (c *controller) ProcessItem(ctx context.Context, key string) error {
-	// Set context deadline for full sync in 10 seconds
-	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	ctx, cancel := context.WithTimeout(ctx, globals.DefaultControllerContextTimeout)
```

Comment:
We're going from 10s to 2 minutes here, which seems like a _long_ time. The issuing controller is just copying things from a Secret + a CertificateRequest into another Secret - why do we want to allow for up to 2 minutes for this? 🤔 I see it mentioned in the proposal doc but can't see the actual rationale for it (maybe I missed it! 😄)